tools/resourcedocsgen/pkg/metadata.go

```diff
@@ -0,0 +1,44 @@
+package pkg
```

Comment:
I defined this in `pkg` thinking that we might have a use for it with https://github.com/pulumi/pulumi/issues/7813 but then maybe not and we'll probably discard/relocate this and use what's in `pulumi/pulumi` if this info becomes available somewhere else.